src/azure-account.ts

```diff
@@ -598,8 +646,8 @@ export async function tokenFromRefreshToken(refreshToken: string, tenantId = com
 async function tokensFromToken(firstTokenResponse: TokenResponse) {
 	const tokenCache = new MemoryCache();
 	await addTokenToCache(tokenCache, firstTokenResponse);
-	const credentials = new DeviceTokenCredentials({ username: firstTokenResponse.userId, clientId, tokenCache });
-	const client = new SubscriptionClient(credentials);
+	const credentials = new DeviceTokenCredentials({ environment: selectedEnvironment, username: firstTokenResponse.userId, clientId, tokenCache });
+	const client = new SubscriptionClient(credentials, selectedEnvironment.resourceManagerEndpointUrl);
```

Comment:
It looks like all our code will need to pass the correct baseUrl when creating a client. That makes the change breaking for dependent extensions. It would make sense to first ship the feature in a hidden way, such that dependent extensions can update and test it.

To do that: Please remove the 'Set Environment' command and instead read from a hidden (i.e., not declared in package.json) setting `azure.experimental.cloud` which cloud to use when signing in.